### ISSUE TITLE
关于gt标签制作的疑问

### ISSUE THREAD

**Issue by DLstudent** [bilylee/SiamFC-TensorFlow#16]
您好，我特地研究了一下construct_gt_score_maps这个函数。制作gt标签的。这个函数接收response的大小即response_size，17*17。然后制作的标签结果全部为Bach_size*17*17的gt map，用来与响应图做交叉熵。不过所有的gt标签，即Bach_size*17*17tensor里面的元素值都为1.0，即全1矩阵。这一点不太理解，为何训练时所有的gt标签都是全1 的tensor？？？

**Comment by TerryYiDa** [comment]
偏离中心 manhattan distance 大于 2 时，标签为什么不是-1呢？

**Comment by PeterHuang2015** [comment]
@bilylee @TerryYiDa 具体实现和原始 siamfc 相比略有改动很正常。。不过话说回来，不是 -1 的话 如 @SMZCC 所讨论的这个[函数](https://github.com/bilylee/SiamFC-TensorFlow/blob/17f51563f081281fc7d966602d393715e7ecb469/metrics/track_metrics.py#L94) 计算的 center error loss 就稍显违和了。。。。centor_score < 0 至少对 response map 的 ground truth 来讲就是不会达到的了:cry:

**Comment by bilylee** [comment]
Hi，

@TerryYiDa 当使用 `tf.nn.sigmoid_cross_entropy_with_logits` 计算 loss 时，输入的 label 只能在 [0, 1] 之间。此时的 label 表示的是当前位置是目标的概率。至于论文中使用的 {-1，+1} 这是表示当前位置的类别，-1 是背景，+1 是目标。当表示类别时，使用什么值并不重要，因为这是一个 categorical value。当 label 是类别时，用 `tf.nn.softmax_cross_entropy_with_logits` 来计算 loss.

@PeterHuang2015 计算 center error loss 时，使用的 response 是 logit，范围一般在 [-10, 10]，此时 center_score < 0 表示当前位置是目标的概率小于 0.5. 是有意义的。

**Comment by PeterHuang2015** [comment]
@bilylee Got it, thx!
